scripts/ci/run_perftest.sh

```diff
@@ -90,24 +125,31 @@ systemctl unmask influxdb.service
 systemctl start influxdb
 
 # Common variables used across all tests
-datestring=$(date +%s)
+datestring=${TEST_COMMIT_TIME}
```

Comment:
I switched to using the unix timestamp of the commit-under-test so:
1. Re-running tests for older commits doesn't make those commits look like the newest change in our results DB
2. We have a consistent randomness seed on a per-commit basis, so if we see something weird happen in a run we can re-run it and expect to get the same generated data & queries